src/containers/CareerDetails.js

```diff
@@ -0,0 +1,13 @@
+import React, { Component } from 'react';
+
+class CareerDetails extends Component {
+  render() {
+    return (
+      <div>
```

Comment:
I'd advise starting as you mean to continue by replacing all these native html elements with `styled components`.